docs/modules/ROOT/pages/_partials/calling-fuse-aggregation-app-endpoint-fail-user-key.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+
+[id='calling-fuse-aggregation-app-endpoint-fail-user-key_{context}']
+= calling-fuse-aggregation-app-endpoint-fail-user-key 
+
+. From the *ActiveDocs* page for the `{fuse-aggregator-app-name}` Application, expand the *GET /flights* endpoint
```

Comment:
Is it necessary to have this line each time?
Would it be enough just the first time as the user would still be on the same page with the Flights API already open?